playbooks/common/openshift-cluster/upgrades/etcd/upgrade.yml

```diff
@@ -36,7 +36,7 @@
       - not openshift.common.is_etcd_system_container | bool
 
     - name: Record containerized etcd version (runc)
-      command: runc exec etcd_container rpm -qa --qf '%{version}' etcd\*
+      command: runc exec etcd rpm -qa --qf '%{version}' etcd\*
```

Comment:
We should read the name from a variable. Given this is not the only occurrence of the literal, it is fine.